packages/@tailwindcss-upgrade/src/template/codemods/theme-to-var.test.ts

```diff
@@ -77,6 +77,13 @@ test.each([
   // Arbitrary property that already contains a modifier
   ['[color:theme(colors.red.500/50%)]/50', '[color:theme(--color-red-500/50%)]/50'],
 
+  // Values that don't only contain `theme(…)` calls should not be converted to
+  // a candidate modifier. Because this is not a color anymore.
```

Comment:
```suggestion
  // Values that don't contain only `theme(…)` calls should not be converted to
  // use a modifier since the color is not the whole value.
```